src/utils/queryUtils.ts

```diff
@@ -0,0 +1,59 @@
+
+interface GraphQueryParameters {
+  /**
+   * List of properties separated by a comma. Properties without a slash are used in $select query parameter. 
+   * Propeties with a slash are used in $expand query parameter.
+   */
+  properties?: string;
+
+  /** Filter expression used in $filter query parameter.*/
+  filter?: string;
+
+  /** If specified then $count=true is included.*/
+  count?: boolean;
+}
+
+export const queryUtils = {
+  /**
+   * Create a query for a request to the Graph API
+   * @param parameters Parameters to be applied to the query.
+   * @returns Query with applied parameters.
+  */
+  createGraphQuery(parameters: GraphQueryParameters): string {
```

Comment:
I'm not sure about this util. I'm wondering if we shouldn't move it to the urlUtil for example. However, discussing the optimal place ,may take some discussion with other maintainers to see where to fit these functions best.

I'd not want to hold this up any longer though. Could we move this back to the command itself for now? Let's create a separate issue where to track this discussion...

Reply:
Same for me, I'm not sure about the right place (util, methods in GraphCommand, etc.). Moved to the command itself.